packages/db/migrations/0105.do.remove_content_report_constraints.sql

```diff
@@ -0,0 +1,13 @@
+-- Type: DO
+-- Name: remove_content_report_constraints
+-- Description: Remove unneeded constraints from the content_report table
+
+BEGIN;
+
+ALTER TABLE omnivore.content_display_report
+    DROP CONSTRAINT content_display_report_page_id_fkey;
+
+ALTER TABLE omnivore.content_display_report
+    DROP CONSTRAINT content_display_report_user_id_fkey;
```

Comment:
Why do we drop this foreign key?

Reply:
@sywhb I think we don't want reports to delete if a user is deleted by default. It makes the tests more complicated, but I think we want to keep those reports.

Reply:
Yeah, you are right. I didn't think about that